RELEASE.md

```diff
@@ -0,0 +1,14 @@
+# Release procedure
+
+This page describes the procedure to release to PyPI.
+
+### Automatic procedure
+
+Create a [Github release](https://github.com/alphacsc/alphacsc/releases) with a version tag (e.g. "v1.2.3").
```

Comment:
should the tag include a `v` and then the version or can we just tag `1.2.3`? If so, what will be the final version  (`v1.2.3` or `1.2.3`)

Reply:
Actually, it does not make any difference. whether the tag is `v1.2.3` or `1.2.3`, the version will be `1.2.3`. I usually prefix version tags with a `v`, but it can be dropped without consequences.

Reply:
Ok thx!